src/main.rs

```diff
@@ -181,6 +181,22 @@ async fn insert_special(
             foundation_account,
             ..
         } => vec![*baker, *foundation_account],
+        SpecialTransactionOutcome::PaydayFoundationReward {
+            foundation_account,
+            ..
+        } => vec![*foundation_account],
+        SpecialTransactionOutcome::PaydayAccountReward {
+            account,
+            ..
+        } => vec![*account],
+        // the following two are only administrative events, they don't transfer to the account,
+        // only to the virtual account.
+        SpecialTransactionOutcome::BlockAccrueReward {
+            ..
+        } => Vec::new(),
```

Comment:
Is this preferred over `vec![]`? Why?

Reply:
`vec![]` should call this eventually. I prefer using it, but it is not a strong preference. Maybe it's easier to type :P